listenbrainz/webserver/static/js/src/personal-recommendations/PersonalRecommendationsModal.tsx

```diff
@@ -0,0 +1,263 @@
+      <div
+        className="modal fade"
+        id="PersonalRecommendationModal"
+        tabIndex={-1}
+        role="dialog"
+        aria-labelledby="PersonalRecommendationModalLabel"
+        data-backdrop="static"
+      >
+        <div className="modal-dialog" role="document">
+          <form className="modal-content">
+            <div className="modal-header">
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+                aria-label="Close"
+                onClick={() => {
+                  this.setState({
+                    users: [],
+                    blurbContent: "",
+                    suggestions: [],
+                  });
+                  // update the followers list
+                  APIService.getFollowersOfUser(currentUser.name)
+                    .then((response) => {
+                      this.setState({ followers: response.followers });
+                    })
+                    .catch((error) => {
+                      this.handleError(error, "Error while fetching followers");
+                    });
+                }}
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 className="modal-title" id="PersonalRecommendationModalLabel">
+                Personally recommend this recording
+              </h4>
+            </div>
+            <div className="modal-body">
+              {users!.map((user) => {
+                return (
+                  <NamePill
+                    title={user}
+                    // eslint-disable-next-line react/jsx-no-bind
+                    closeAction={this.removeUser.bind(this, user)}
+                    closeButton
+                  />
+                );
+              })}
+              <input
+                type="text"
+                className="form-control"
+                onChange={this.searchUsers}
+                placeholder="Search and add users"
+              />
+              <SearchDropDown
+                suggestions={suggestions}
+                // eslint-disable-next-line react/jsx-no-bind
+                action={this.addUser}
+              />
+              <p style={{ marginTop: "24px", marginBottom: "24px" }}>
+                Tell the users selected above why are you recommending them{" "}
+                <b>
+                  {" "}
+                  {track_name} by {artist_name}
+                </b>
+                ? (Optional)
+              </p>
+              <div className="form-group">
+                <textarea
+                  className="form-control"
+                  id="blurb-content"
+                  placeholder="Let your followers know why you recommended this song"
+                  value={blurbContent}
+                  name="blurb-content"
+                  rows={4}
+                  style={{ resize: "vertical" }}
+                  spellCheck="false"
```

Comment:
Y u no specllheck?

:p 
Jokes aside, what is the rationale for disabling this?

Reply:
Hi, sorry for the late reply as I was busy conducting Software Freedom Day event :)

I disabled spellcheck because other modals have it disabled (Pin Recording Modal and CB Review Modal)
I did think if it's needed or not, but then I realized that I often use "Hinglish" (Hindi but with latin script) which might come off as wrong in the textbox.

Reply:
Hm, interesting . Didn't realize that was the case on the pin modal.
I tentatively think we shouldn't disable it (for either modal, but let's focus on this one for this PR).
If we're struggling to find a rationale for disabling it it's probably best to keep browser defaults that users would expect.